frontend/src/assets/projects.js

```diff
@@ -183,6 +183,41 @@ const projects = [
       }
     ],
     website: 'https://dev.codeforpoznan.pl/'
+  },
+  {
+    badge: 'Aktywny',
+    description:
+      'Watchdog Polska od 2003 roku stoi na straży prawa do informacji w Polsce. Postrzegają je nie tylko jako warunek dobrego państwa, ale przede wszystkim jako jedno z praw człowieka, które chroni ludzką godność, daje wolność wyrażania opinii i bezpieczeństwo przed nadużyciami władzy.W ciągu 17 lat swojej działalności Sieć Obywatelska Watchdog Polska uczestniczyła w około 1122 sprawach sądowych dotyczących prawa do informacji. Niektóre z nich miały przełomowe znaczenie dla społeczeństwa. Razem współtworzymy system służący do usprawnienia obiegu dokumentów Stowarzyszenia, w szczególności w zakresie obiegu korespondencji spraw sądowych oraz archiwizacji dokumentacji.',
```

Comment:
Not sure if it was copy-pasted or invented here but i would suggest some minor improvements in the description:
'Watchdog Polska od 2003 roku stoi na straży prawa do informacji. Rozumiane jest ono nie tylko jako warunek funkcjonowania dobrego państwa, ale przede wszystkim jako jedno z praw człowieka, które chroni ludzką godność, daje wolność wyrażania opinii i zabezpiecza przed nadużyciami władzy. W ciągu 17 lat swojej działalności Sieć Obywatelska Watchdog Polska uczestniczyła w około 1122 sprawach sądowych dotyczących prawa do informacji. Niektóre z nich miały przełomowe znaczenie dla społeczeństwa. Razem współtworzymy system służący do usprawnienia obiegu dokumentów Stowarzyszenia, w szczególności w zakresie korespondencji oraz archiwizacji.'

Reply:
Not sure either. Thanks for catching that. I adjusted text to your suggestions, except the ending I made myself another correction there.